c2corg_api/views/outing.py

```diff
@@ -31,6 +33,37 @@
     validate_associations, OUTING_TYPE, False)
 
 
+def validate_dates(request, **kwargs):
+    utc_now = datetime.datetime.now(datetime.timezone.utc)
+    utc_now_plus_12h = (utc_now + datetime.timedelta(hours=12)).date()
+
+    document = request.validated.get('document')
```

Comment:
`request.validated` IS the document :)

```suggestion
    document = request.validated
```

Reply:
When executing the unit-tests, the document is indeed in `request.validated`. But from the node application `c2c_ui`, the document is inside `request.validated['document']`. I am pretty sure this is not an expected situation. Anyway, this is temporary but 6b32b77 handles both cases.

Reply:
...checking